### ISSUE TITLE
Be more helpful

### ISSUE THREAD

**Issue by jonallured** [jonallured/danger-commit_lint#10]
As @amatriain points out in feedjira/feedjira#366, Commit Lint doesn't do anything to help one fix what has been marked as broken.

I think I've taken it for granted that people know about amending messages and interactive rebasing, not to mention that force pushing their branch is totally appropriate. What I'd like to see is a write up of some ways you can fix a violation and then when the bot comments, the comment should link to this write up.

**Comment by jonallured** [comment]
Ok, I've taken a stab at documenting some tips for fixing commit messages in violation, now I just need to figure out the right way to add a link to this in the feedback Danger adds to the PR. I don't think it belongs in the messages added by `messaging.warn` or `messaging.error`, so maybe I need to use `messaging.message`? Does that sound right @orta?

And I guess I'd need to check to see if there are any violations so that I only print the message when necessary - not sure if I have to do that or if there's something Danger provides plugin authors.

**Comment by orta** [comment]
Yeah,`markdown` is really good for this, would recommend maybe throwing "fix recommendation strings" into a set and then using `markdown` at the end.

I don't know if anyone has experimented with `<detail>`  [inside an existing warning](https://gist.github.com/orta/057eab0832773aa299a6ebd39187ab06)

But that could be a good pattern for one off errors
